static/src/javascripts/bootstraps/standard/main.js

```diff
@@ -110,6 +110,27 @@ define([
             }
         });
 
+        /*
+         *  Interactive bootstraps.
+         *
+         *  Interactives are content, we want them booting as soon (and as stable) as possible.
+         */
+
+        if (
+            config.switches.bootInteractivesFromMain &&
+            /Article|Interactive|LiveBlog/.test(config.page.contentType)
+        ) {
+            $('figure.interactive').each(function (el) {
```

Comment:
As far as `ismodern` goes, Interactives should run on all browsers, we cannot have blank pages on older browsers. @wpf500 is it the case interactives only work on newer "modern" browsers?